1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -1,25 +1,21 @@
 
 # Polyfills
 
-The JavaScript language steadily evolves. New proposals to the language appear regularly, they are analyzed and, if considered worthy, are appended to the list at <https://tc39.github.io/ecma262/> and then progress to the [specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
+The JavaScript language steadily evolves. New features requests are regularly [proposed](https://github.com/tc39/proposals) to the language and are vetted through a [4 part staging process](https://tc39.github.io/process-document/). If considered worthy, these are added to the evergreen spec at <https://tc39.github.io/ecma262/> and then get standardized to the [yearly release specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
 
-Teams behind JavaScript engines have their own ideas about what to implement first. They may decide to implement proposals that are in draft and postpone things that are already in the spec, because they are less interesting or just harder to do.
-
-So it's quite common for an engine to implement only the part of the standard.
```

Comment:
I don't think this is true, or at least in the way you are describing. All engines implement the spec, they just take time to do so. The reason why they don't implement something is because they decide it's too slow to do (and usually something that hasn't gone through the Staging process). Now TC39 uses Stage 3 to make sure that 2 implementations have shipped so that there isn't this problem

Reply:
Sure, I'm aware of the proposals flow. With your update the description is better.